src/java/org/apache/cassandra/net/ResponseVerbHandler.java

```diff
@@ -49,7 +49,7 @@ public void doVerb(MessageIn message, int id)
         else
         {
             //TODO: Should we add latency only in success cases?
```

Comment:
Do you think we should remove this TODO while are here?

Reply:
What do you think we should record as the latency under a failure case? Timeouts are already handled elsewhere, so we're just left with execution failures. A failure counting as a timeout seems the most reasonable choice (as nodes that are throwing failures would very rapidly be de-preferred), but not sure.

Reply:
Let's just leave it alone the scope of this is already big enough.

Reply:
+1, ack